libs/arcade-cli/arcade_cli/main.py

```diff
@@ -69,6 +70,14 @@
 console = Console()
 
 
+def handle_error_with_debug(error: Exception, message: str, debug: bool) -> None:
```

Comment:
might as well call this ``handle_cli_error(message: str, error: Exception | None = None, debug: bool = True, exit : bool = True``

would get rid of alot of typer.exits()